CONTRIBUTING.md

```diff
@@ -55,4 +69,157 @@ We prefer all communications to be in English.
 
 Microsoft follows the principle of [Coordinated Vulnerability Disclosure](https://www.microsoft.com/en-us/msrc/cvd).
 
-<!-- END MICROSOFT SECURITY.MD BLOCK -->
\ No newline at end of file
+<!-- END MICROSOFT SECURITY.MD BLOCK -->
+
+# Develop for CBL-Mariner
+## Choose a repo
+The first step in developing with CBL-Mariner is to determine which repo fits your use case. If you are just looking to develop using CBL-Mariner's tools and packages, check out the [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials) repo. `CBL-MarinerTutorials` is geared towards creating custom CBL-Mariner builds rather than contributing to the tools or common packages. 
```

Comment:
Recommend to start everyone looking to contribute packages over at CBL-MarinerTutorials, rather than presenting a decision first.

Once they have a package building, then they can read on to determine if contributing that package into this CBL-Mariner repo makes sense.